xml/System/IntPtr.xml

```diff
@@ -244,23 +244,25 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>A platform-specific type that is used to represent a pointer or a handle.</summary>
+    <summary>Represents a signed integer where the bit-width is the same as a pointer.</summary>
```

Comment:
The format used by the other integer types is:
> Represents a 64-bit signed integer.

This would then correspond, technically speaking, to:
> Represents a ptr-bit signed integer.

However, I don't think that is necessarily as clear. I've found people sometimes understand it instantly and other times do not, when it's described that way.